### DISCUSSION TITLE
New feature

### DISCUSSION THREAD

**Discussion by zenorach** [NickvisionApps/Denaro#804] [Ideas]
Ok if I'm an idiot let me know, but tried searching and messing with the program I can't find a way to add multiple items to the same group all at once INSIDE the GUI.

Use case:
I have a bucket for "Mandatory bills" and tags for tracking individual portions. I'd like a easy way via the GUI to select save 15 items and add them to the "Mandatory bills group" Then maybe go into that group later and select the 5 in there that are all related to XYZ and add the Tag XYZ to just those.

Perhaps clicking on the ID # itself

**Comment by nlogozzo** [comment]
It is not a feature, yet, but there is already a request for it:
https://github.com/NickvisionApps/Denaro/issues/502

**Reply by zenorach** [reply]
.... well I feel dumb now. Sorry about that. I couldn't find it.
Now. Next dumb question. What is the correct way in your community to promote this and see how big a deal it is to people? because if I'm in the minority I don't wanna be a pest.

**Reply by nlogozzo** [reply]
No issue / person is ever a pest!

The fact that this issue already exists shows that there's interest in it!

Little by little we will implement and complete these issues...this one in particular is something i'd like myself
Just finished up my final semester now, so it's been slow, but at the end of May i'll be back full time working on everything!

**Reply by zenorach** [reply]
First off good for you for working on this during your schooling. Second thank you, that's good to hear. Third. I've never coded a visual program for linux (lot of parsers and monitors for sysadmin work, but nothing visual).
You got an idiot's guide to spinning up you've used in the past that meshes well with your existing framework? Maybe I can take a crack and push a potential solution.

**Reply by nlogozzo** [reply]
> You got an idiot's guide to spinning up you've used in the past that meshes well with your existing framework? 

Not really, especially since I am currently rewriting everything from the current C# implementation to a C++ implementation (see #784 for progress)...once I finish porting everything over, then I will work on adding in new features.

So I'd say wait for now until I get the C++ rewrite merged and then we can talk about getting you set up to contribute as well! (Even something like where you implement everything in the backend and I'll get it wired up in the GUI could work!)
